### DISCUSSION TITLE
[globalize] package default export is not callable

### DISCUSSION THREAD

**Discussion by carlpaten** [DefinitelyTyped/DefinitelyTyped#57103] [Issues with a @types package]
I've produced a minimal repro [here](https://github.com/lilred/globalize-type-bug/blob/master/index.ts), using the example code provided [here](https://github.com/globalizejs/globalize/blob/master/README.md#getting-started). Unfortunately I'm not sure how to fix.

[a comment by an automated account is not shown]

**Comment by soerenBoisen** [comment]
I have the same problem. Came up with this hack:
```typescript
        // HACK: For some reason TS will not allow me to call Globalize as function despite the declaration file
        // See: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/57103
        const globalize = Globalize as unknown as Globalize.Static
```
